settings.gradle.kts

```diff
@@ -13,7 +13,6 @@ gradleEnterprise {
     buildScan {
         termsOfServiceUrl = "https://gradle.com/terms-of-service"
         termsOfServiceAgree = "yes"
-        publishAlways()
```

Comment:
Why do you remove it?

Reply:
why publishing the scan on every IDE sync?

Reply:
It was a way to follow the failure tests because the output was too verbose previously.
You fixed it with your gradle plugin.

So the question is more: Is there a good use case for gradle scan or should we remove it?

Reply:
Build scan is still useful, so I would keep it in CI, and it might be useful for local runs.
However, I do not really see value in publishing scans from all the actions like IDE sync, `gradlew tasks` and so on.

Reply:
Got it. We just need to add it somewhere in a github actions pipeline, right?

Reply:
That's the plan: https://github.com/cbeust/testng/pull/2584/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R70

Reply:
It missed it 👍